integration_tests/accessibility/accessibility.spec.js

```diff
@@ -0,0 +1,85 @@
+import { getRecallResponse, getAllFieldsHistoryResponseJson } from '../mockApis/mockResponses'
+
+const pa11yArgs = { runners: ['axe'], standard: 'WCAG2AA' }
+const nomsNumber = 'A1234AA'
+const recallId = '123'
+const urls = [
+  '/',
```

Comment:
maybe worth adding:
- confirmation page (for either book, assess or dossier)
- user details
- check dossier details (ie first page after you click Create dossier)
- change history (documents tab)
- change history for a document
- last known address page

Reply:
We've added all the above except the change history (for documents) ones as we couldn't figure out how to get the mocks to work... Maybe add this in then pair up to get those (and checking error states) in?